docs/src/Groups/matgroup.md

```diff
@@ -79,8 +79,8 @@ pol_elementary_divisors(A::MatElem{T}) where T
 generalized_jordan_block(f::T, n::Int) where T<:PolyRingElem
 generalized_jordan_form(A::MatElem{T}; with_pol=false) where T
 matrix(A::Vector{AbstractAlgebra.Generic.FreeModuleElem{T}}) where T <: RingElem
-upper_triangular_matrix(L)
-lower_triangular_matrix(L)
+upper_triangular_matrix(L::AbstractVector{T}) where {T <: NCRingElement}
+lower_triangular_matrix(L::AbstractVector{T}) where {T <: NCRingElement}
```

Comment:
Honestly I would just drop them from the documentation here. I am not sure why we listed them here (in the matrix group section) in the first place. 
```suggestion
```

Actually almost everything in this "Utilities for matrices" section is questionable here. Some should perhaps also be moved. Some should not be documented at all (I don't even understand the docstring for some of them, e.g. `pol_elementary_divisors`)

(This comment is not meant to be a blocker for this PR)